packages/client/src/app/services/network.service.ts

```diff
@@ -0,0 +1,1450 @@
+        }
+
+        return this.handleResponse(
+            this.http.get<PaginateResult<ContentModel>>(route, { observe: 'response', withCredentials: true }),
+            null,
+            err => {
+                //this.snackBar.open(`Something went wrong fetching this content. Try again in a little bit.`);
+            }
+        );
+    }
+
+    /**
+     * Fetches one section from the API using the provided `sectionID`.
+     * 
+     * @param sectionId The section ID
+     */
+    public fetchSection(sectionId: string): Observable<Section> {
+        return this.handleResponse(
+            this.http.get<Section>(`${this.baseUrl}/content/sections/fetch-one-by-id?sectionId=${sectionId}&published=true`, { observe: 'response', withCredentials: true }),
+            null,
+            err => {
+                //this.snackBar.open(`Something went wrong fetching this section. Try again in a little bit.`);
+            }
+        );
+    }
+
+    /**
+     * Sets a user's rating to Liked.
+     * 
+     * @param setRating Information to set the new rating
+     */
+    public setLike(setRating: SetRating): Observable<ReadingHistory> {
+        return this.handleResponse(
+            this.http.patch<ReadingHistory>(`${this.baseUrl}/content/set-like`, setRating, { observe: 'response', withCredentials: true }),
+            null,
+            err => {
+                //this.snackBar.open(err.error.message);
+            }
+        );
+    }
+
+    /**
+     * Sets a user's rating to Disliked.
+     * 
+     * @param setRating Information to set the new rating
+     */
+    public setDislike(setRating: SetRating): Observable<ReadingHistory> {
+        return this.handleResponse(
+            this.http.patch<ReadingHistory>(`${this.baseUrl}/content/set-dislike`, setRating, { observe: 'response', withCredentials: true }),
+            null,
+            err => {
+                //this.snackBar.open(err.error.message);
+            }
+        );
+    }
+
+    /**
+     * Sets a user's rating to NoVote.
+     * 
+     * @param setRating Information to set the new rating
+     */
+    public setNoVote(setRating: SetRating): Observable<ReadingHistory> {
+        return this.handleResponse(
+            this.http.patch<ReadingHistory>(`${this.baseUrl}/content/set-no-vote`, setRating, { observe: 'response', withCredentials: true }),
+            null,
+            err => {
+                //this.snackBar.open(err.error.message);                
+            }
+        );
+    }
+    //#endregion
+
+    //#region ---COVER ART & AVATARS---
+
+    /**
+     * Uses the given uploader to an image up to the server. Attempts to cast
+     * the server's response into a T and return it.
+     * 
+     * @param uploader The file uploader, prefilled with the URL and instructions for uploading the image.
+     */
+    public changeImage<T extends FrontendUser | ContentModel>(uploader: FileUploader): Observable<T> {
```

Comment:
Finally get to consolidate this method into one place instead of repeating it three different times \o/